src/types.tsx

```diff
@@ -161,6 +161,17 @@ export interface ScreenProps extends ViewProps {
    * @platform ios
    */
   homeIndicatorHidden?: boolean;
+  /**
+   * If a screen is marked `hidden`, the resources will be retained, but it will not be rendered
+   * in the stack. This is used when preloading screen but can also be used as a way to support
+   * iOS picture-in-picture, where you have a video overlay originating on a screen, and even
+   * when popping the screen off the stack the resources need to be retained (not garbage collected),
+   * so that the video can continue playing. Additionally, this allows the screen to be "un-hidden"
+   * (visually, pushed back onto the stack) when the user taps on the overlay video.
+   *
+   * Only supported on iOS and Android.
```

Comment:
If it is supported on `Android`, should the code for `Android` end up in this PR?

Reply:
I would like to add that to this PR once I'm sure the iOS side works as expected. But it could be split up if you want.

Reply:
I think the PR is pretty small so its ok to have both platforms in one PR.